etc/utils.py

```diff
@@ -0,0 +1,64 @@
+import random
+import re
+from enum import Enum
+
+from etc.consts import WORDS_FILE_NAME, VALID_WORD_RX
+
+
+class Color(Enum):
+    GREY = 0
+    YELLOW = 1
+    GREEN = 2
+
+
+def get_random_word(words):
+    """
+    Given a list of words, returns a random word.
+
+    :param words: list of words (strings)
+    :return: a random word from words list
+    """
+    if not words:
+        raise Exception("Words list is empty!", words)
+
+    rand_idx = random.randint(0, len(words)-1)
+    return words[rand_idx]
+
+
+def read_words():
```

Comment:
I dont like this name, most of the time `read` functions get a param to read from.
You must F12 on this function to understand what it does.
Furthermore this function is an util function so it should be generic as posible.
I think this function should be `def read_word_from_file(file_path)`